src/msw/window.cpp

```diff
@@ -6303,6 +6303,8 @@ MSWInitAnyKeyEvent(wxKeyEvent& event,
     event.m_rawCode = (wxUint32) wParam;
     event.m_rawFlags = (wxUint32) lParam;
     event.SetTimestamp(::GetMessageTime());
+
+    event.m_isRepeat = ((HIWORD(lParam) & KF_REPEAT) == KF_REPEAT;
```

Comment:
I didn't notice it, but luckily the CI builds did: there is an extra parenthesis here:
```suggestion
    event.m_isRepeat = (HIWORD(lParam) & KF_REPEAT) == KF_REPEAT;
```